src/Resources/views/dashboard/cards.html.twig

```diff
@@ -1,4 +1,4 @@
-<div class="flex flex-col bg-neutral-300 drop-shadow">
+<div class="flex flex-col justify-between bg-neutral-300 drop-shadow">
```

Comment:
Nit: The more explicit and future-proof solution might be giving the button a `margin-block-start: auto` instead of trying to fix this via the parent. Because this solution might break if we add more child elements.